app/src/main/java/io/github/droidkaigi/confsched/util/DateUtil.java

```diff
@@ -35,8 +35,14 @@ public static String getMonthDate(Date date, Locale locale, Context context) {
     }
 
     @NonNull
-    public static String getHourMinute(Date date) {
-        return String.valueOf(DateFormat.format(FORMAT_KKMM, date));
+    public static String getHourMinute(Date date, Context context) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
+            Locale locale = AppUtil.getCurrentLocale(context);
+            String pattern = DateFormat.getBestDateTimePattern(locale, FORMAT_KKMM);
+            return new SimpleDateFormat(pattern, locale).format(date);
```

Comment:
If you use `DateFormat.getBestDateTimePattern()`, time style are changed by locale.
<img width="779" alt="2016-02-09_21_32_52_png_and_2016-02-08_18_19_46_png" src="https://cloud.githubusercontent.com/assets/1269214/12938763/0c4cfec8-cffb-11e5-8b15-35f4f42daa97.png">